docs/docs-content/enterprise-version/install-palette/palette-management-appliance.md

```diff
@@ -20,6 +20,10 @@ used to boot the nodes, which are then clustered to form a Palette management cl
 Once Palette has been installed, you can download pack bundles to create your cluster profiles. You will then be able to
 deploy clusters in your environment.
 
+There is an additional option to download and install the Third Party pack that contains complementary functionality to
+Palette. This pack is not required for Palette to function, but it does provide additional features and capabilities
+that enhance the Palette experience.
```

Comment:
@Eulerizeit @tiwarisumit3 Can you review this?

I have kept this fairly generic, but feel free to suggest any specific wording regarding the third-party pack functionality.

Reply:
The relevant information is now included with links to relevant doc pages.